packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="flex flex-col gap-y-12">
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Quick start</h2>
```

Comment:
Later this will be dismissible and the content might change, but there is no design for that yet